day_7/high_level.md

```diff
@@ -0,0 +1,28 @@
+# My Notes on the Caesar Cipher
+
+The Caesar Cipher is a means to encrypt a message. It takes the alphabet and your given number, and skews the alphabet N amount of times to the right in relation to our original alphabet.
+
+## Our pseudocode below:
+
+We can use a method to capture user data.
+
+Let's make a function that will take two parameters: the string that the user wants to encrypt, and the number of degrees we will shift our alphabet to the left.
+
+`def caesar_cipher(string, degrees)`
```

Comment:
Nice work making a clear step-by-step document with some code snippets to illustrate the explanation of how you'll go about making a caesar cipher.